api/controllers/v1/jobs.ts

```diff
@@ -11,13 +11,25 @@ import { Job, JobStatus } from '../../../src/entities/job';
 import { ECSContainer } from '../../../src/services/containerServices';
 import { SQSConnector } from '../../../src/services/queue';
 import { Batch } from '../../../src/services/batch';
+import { APIGatewayEvent, APIGatewayProxyResult, SQSEvent, SQSRecord } from 'aws-lambda';
 
-export const TriggerLocalBuild = async (event: any = {}, context: any = {}): Promise<any> => {
+export const TriggerLocalBuild = async (event: APIGatewayEvent): Promise<APIGatewayProxyResult> => {
   const client = new mongodb.MongoClient(c.get('dbUrl'));
```

Comment:
Is this connection ever used? (I know this doesn't have to do with your PR, but great time to clean this if so)

Reply:
Good catch! It is not used. I will remove.